Improve transition from Fundamentals -> induction (#293)
CodeYourFuture/curriculum: content/fundamentals/blocks/next-steps/index.md

+++
title = 'Next steps 👣'
headless = true
time="20"
facilitation = false
emoji= '🧩'
[objectives]
    1='Figure out what we need to do next in the course'
    2='Locate the setup and prep view on the curriculum'
    3='Practice asking for guidance/help when we are stuck'
+++

## For trainees

In this section, you have a set of questions/tasks to orient you towards what is coming next in the course.

> Get into pairs with somebody you've not paired up with today and try figuring out the answers/solutions to the following questions/tasks.

Reviewer: ```suggestion
Pair up with a new person. In pairs, complete the following tasks:
```
